File: tests/test-services/typescript/template/src/services.ts
import {echoService} from './echo_service'
import {checkService} from './check_service'
import {echoService as echoServiceV2} from './v2/echo_service'

export const services = {echoService: echoService(), checkService: checkService(), echoServiceV2: echoServiceV2()}

File: test-services/go/template/services/services.go
package services

import (
	v2 "{{project.value}}/services/v2"
	"{{project.value}}/spec"
)

func Create() spec.Services {
	echoServiceV2 := &v2.EchoService{}
	echoService := &EchoService{}
	checkService := &CheckService{}
	return spec.Services{echoServiceV2, echoService, checkService}
}
